### PR TITLE
Update testing/mm testcase, add stress test

### PR DESCRIPTION
**Pull request by GUIDINGLI**
## Summary
Update testing/mm testcase, add stress test

### CODE REVIEW

File: testing/mm/Kconfig [hunk 1]
```diff
@@ -20,7 +20,7 @@ config TESTING_MM_PROGNAME
 
 config TESTING_MM_PRIORITY
 	int "Task priority"
-	default 100
+	default 250
```
Comment (acassis) [inline testing/mm/Kconfig:23]: Why do you need to increase the priority? Does it fail when using priority 100?

**Comment by xiaoxiang781216** [comment]
Fix here: https://github.com/apache/incubator-nuttx/pull/4051

File: testing/mm/Kconfig (hunk 1 above)
Comment (xiaoxiang781216) [reply testing/mm/Kconfig:23]: We run memtest in the background. The high prirority could avoid the bad high prority task block memtest.

Comment (acassis) [reply testing/mm/Kconfig:23]: Ok! BTW it still be possible to have higher priority tasks (251-255)

Comment (xiaoxiang781216) [reply testing/mm/Kconfig:23]: Yes, but it's rare:).

**Review by yamt — COMMENTED** [review]
why do you add this to testing/mm, rather then introducing a new command?
do you plan to add more and more later?

**Comment by xiaoxiang781216** [comment]
Let's close this PR.

**Pull request closed without merging**